randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -267,6 +272,10 @@ Extra - room_id: [6]
               After Sector 4 Water Level Lowered
               # Damage Reqs
               Gravity Suit and Damage Runs (Beginner) and Damage Runs (Intermediate) and Electrified Water Damage ≥ 20 and Electrified Water Damage ≥ 80
+          Any of the following:
```

Comment:
I thought the upper bomb block doesnt respawn?

Reply:
![image](https://github.com/user-attachments/assets/37904e80-cd67-414d-9b54-f399e9f8f314)
circle blocks reform, crossed out ones do not

Reply:
This has just been merged into main on MARS repo. None off these blocks will respawn now:
https://github.com/MetroidAdvRandomizerSystem/MARS-Fusion/pull/95

Reply:
only on a certain configuration tho, which we'll assume here to be off for safetyness

Reply:
Ah, I forgot about the anti-softlock flag. You're correct here, my bad.